src/cryptojwt/key_bundle.py

```diff
@@ -254,6 +255,8 @@ def __init__(
             self.httpc = requests.request
 
         self.httpc_params = httpc_params or {}
+        if not self.httpc_params.get("timeout"):
+            self.httpc_params["timeout"] = DEFAULT_HTTPC_TIMEOUT
```

Comment:
instead or copy the same code all around probably it would be better to create a default httpc loader and use it in all the classes